examples/grafana-agent/grafana-provisioning/dashboards/main.json

```diff
@@ -0,0 +1,82 @@
+{
+  "annotations": {
+    "list": [
+    ]
+  },
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 1,
+  "links": [],
+  "liveNow": false,
+  "panels": [
+    {
+      "datasource": {
+        "type": "phlare",
```

Comment:
reference #5: this is a tricky one, there's a PR here https://github.com/grafana/grafana/pull/67847